pkg/health/report.go

```diff
@@ -0,0 +1,37 @@
+package health
+
+import (
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/cloud-run-release-operator/pkg/config"
+)
+
+// StringReport returns a human-readable report of the diagnosis.
+//
+// The returned string has the format:
+// status: healthy
+// metrics:
+// - request-latency[p99]: 500.00 (needs 750.0)
+// - request-count: 800 (needs 1000)
```

Comment:
this will most likely go out of date, I'd refrain from listing it here, but it's ok